package.json

```diff
@@ -24,22 +25,24 @@
     "bitcoin-core": "2.0.0",
     "bitcoinjs-lib": "5.0.3",
     "body-parser": "~1.19.0",
-    "cookie-parser": "^1.4.4",
+    "cookie-parser": "^1.4.5",
```

Comment:
I know it happened automatically, but please change all of the updated dependencies back to the previous versions